config/config_test.go

```diff
@@ -411,33 +414,21 @@ func TestConfig_ParserInterfaceNewFormat(t *testing.T) {
 			param: map[string]interface{}{
 				"HeaderRowCount": cfg.CSVHeaderRowCount,
 			},
-			mask: []string{"TimeFunc", "Log"},
-		},
-		"json_v2": {
-			mask: []string{"Log"},
+			mask: []string{"TimeFunc"},
 		},
 		"logfmt": {
 			mask: []string{"Now"},
 		},
-		"xml": {
-			mask: []string{"Log"},
-		},
-		"xpath_json": {
-			mask: []string{"Log"},
-		},
-		"xpath_msgpack": {
-			mask: []string{"Log"},
```

Comment:
Help me understand why we can just drop the json_v2, XML, xpath_json, and xpath_msgpack tests?

Reply:
We don't drop the tests here, we just remove the extra-masking of the Logger during comparison. This is possible because I now mask the `telegraf.Logger` field(s) in general (see line 510), therefore extra masking is not necessary any longer.
The list of parsers to test is defines in line 377ff.